beater/beater.go

```diff
@@ -44,16 +43,19 @@ func (bt *beater) Run(b *beat.Beat) error {
 	}
 
 	server := newServer(bt.config, callback)
-	start(server, bt.config.SSL)
+	bt.errCh = start(server, bt.config.SSL)
 	defer stop(server)
 
 	logp.Info("apm-server is running! Hit CTRL-C to stop it.")
```

Comment:
`start` happens in L46, and log in L49. 
Before the refactor merged yesterday that was indeed the case, but not anymore.